.github/workflows/main.yml

```diff
@@ -78,9 +78,9 @@ jobs:
           echo "-={ Step 4: pylint }=-"
           if [ "${{ matrix.python-version }}" != "3.6.7" ]; then
             PYTHONPATH=.:$PYTHONPATH pylint --rcfile=.pylintrc $python_file || true
```

Comment:
Please remove this line

Reply:
Removed in https://github.com/nrpy/nrpy/pull/15/commits/94e603332b922c2e543a2aaa9ea5a63af68bdfa8.